i18n/en/pop_desktop_widget.ftl

```diff
@@ -3,13 +3,13 @@ action-applications-description = Pressing the Super key opens the Applications
 action-launcher = Launcher
 action-launcher-description = Pressing the Super key opens the Launcher
 action-workspaces = Workspaces
-action-workspaces-description = Pressing the Super key opens the Window and Workspaces Overview
+action-workspaces-description = Pressing the Super key opens the Windows and Workspaces Overview
 
 click-action-cycle = Launch or Cycle Windows
 click-action-minimize = Launch or Minimize Windows
 click-action-minimize-or-previews = Launch, Minimize, or Preview Windows
 
-date-combo = Date & Time and Notifications Position
+date-combo = Calendar and Notifications Position
```

Comment:
It is jarring to read "&" followed by "and". I suggest renaming "Date & Time" to "Calendar" to make this sentence much simpler.

Alternatively, it could be renamed to "Date, Time, and Notifications Position".